Sources/Sentry/SentryProfiler.mm

```diff
@@ -408,16 +405,12 @@ + (SentryEnvelopeItem *)createProfilingEnvelopeItemForTransaction:(SentryTransac
     };
 
     // add the serialized info for the associated transaction
-    const auto firstSampleTimestamp = slicedSamples.firstObject.absoluteTimestamp;
-    const auto profileDuration = getDurationNs(firstSampleTimestamp, getAbsoluteTime());
-
-    const auto transactionInfo = [self serializeInfoForTransaction:transaction
-                                                   profileDuration:profileDuration];
+    const auto transactionInfo = [self serializeInfoForTransaction:transaction];
     if (!transactionInfo) {
         SENTRY_LOG_WARN(@"Could not find any associated transaction for the profile.");
         return nil;
     }
-    payload[@"transactions"] = @[ transactionInfo ];
```

Comment:
This is the entirety of the linked transaction info schema change. Since profiles are now 1:1 with profiles again, we don't have to send an array. And the key name changed plural to singular.